homeassistant/components/sonos/entity.py

```diff
@@ -57,10 +57,7 @@ async def async_will_remove_from_hass(self) -> None:
     async def async_fallback_poll(self, now: datetime.datetime) -> None:
         """Poll the entity if subscriptions fail."""
         if not self.speaker.subscriptions_failed:
-            if soco_config.EVENT_ADVERTISE_IP:
-                listener_msg = f"{self.speaker.subscription_address} (advertising as {soco_config.EVENT_ADVERTISE_IP})"
-            else:
-                listener_msg = self.speaker.subscription_address
+            listener_msg = f"{self.speaker.subscription_address} (advertising as {soco_config.EVENT_ADVERTISE_IP})"
```

Comment:
This changes the logic a bit. The config option does not necessarily need to be set.

Reply:
I believe the change here can be reverted completely without issues.

Reply:
How's this?

Reply:
Why not keep the old code? Was it causing an issue?

Reply:
Per @jjlawren comment, it seemed as this change was ok but we can definitely set that one aside from this PR

Reply:
Agree that it doesn't relate to typing and should probably be reverted as it was.